lib/Dhalang/puppeteer.rb

```diff
@@ -5,16 +5,17 @@ class Puppeteer
         private_constant :NODE_MODULES_PATH
 
         USER_OPTIONS = {
-                navigationTimeout: 10000,
-                navigationWaitUntil: 'load',
-                navigationWaitForSelector: '',
-                navigationWaitForXPath: '',
-                userAgent: '',
-                isHeadless: true,
-                viewPort: '',
-                httpAuthenticationCredentials: '',
-                isAutoHeight: false,
-                chromeOptions: []
+            navigationTimeout: 10000,
+            printToPDFTimeout: 0, # unlimited
```

Comment:
The previous version of Puppeteer doesn't have a timeout during a 'printToPDF' call.